File: Java/1774 - Capitalization.java
import java.util.Scanner;

public class Capitalization {
    
    public static void main(String[] args){
        Scanner in = new Scanner(System.in);
        String str = in.next();
        boolean state  = false;
        
        for(int i = 1; i < str.length(); i++){
            if(Character.isUpperCase(str.charAt(i))){
                state = true;
            }
        }
        if(state == false){
            System.out.println(str);
        } else {
            String x = "";
            for(int i = 0; i < str.length(); i++){
                if(Character.isUpperCase(str.charAt(i))){
                    x += Character.toLowerCase(str.charAt(i));
                }else {
                    x += Character.toUpperCase(str.charAt(i));
                }
            }
            System.out.println(x);
        }
    }
            
}

File: c++/1774 - Capitalization.cpp
#include <iostream>
#include <iomanip>
#include <algorithm>
#include <cmath>
#include <string>
#include <vector>

using namespace std;

bool isLowerCase(char c)
{
    return (c >= 'a' && c <= 'z');
}

bool isUpperCase(char c)
{
    return (c >= 'A' && c <= 'Z');
}

char toLowerCase(char c)
{
    if (isUpperCase(c))
        return c + ('a' - 'A');
    return c;
}

char toUpperCase(char c)
{
    if (isLowerCase(c))
        return c - ('a' - 'A');
    return c;
}

int main()
{
    string str;
    cin >> str;

    bool capsLockByAccident = false;
    for (int i = 1; i < str.length(); ++i)
    {
        if (isUpperCase(str[i]))
        {
            capsLockByAccident = true;
            break;
        }
    }

    if (capsLockByAccident)
    {
        for (int i = 0; i < str.length(); ++i)
        {
            str[i] = isUpperCase(str[i]) ? toLowerCase(str[i]) : toUpperCase(str[i]);
        }
    }
    cout << str << endl;
}
